tests/integration/smoke_test.go

```diff
@@ -48,7 +48,7 @@ func (suite *SmokeSuite) SetupSuite() {
 
 	suite.installer = installer.NewK8sRookhelper(kh.Clientset, suite.T)
 
-	isRookInstalled, err := suite.installer.InstallRookOnK8s(suite.namespace, "bluestore")
+	isRookInstalled, err := suite.installer.InstallRookOnK8sWithHostPathAndDevices(suite.namespace, "bluestore", "", true, 3)
```

Comment:
want to add a comment to the suite level that this test will exercise devices?

do we have any coverage for filestore on device? or only bluestore now?

Reply:
yes, i'll add comments for the tests to start documenting what the tests do.
devices are now only tested for bluestore. It will complicate the device contention if we add another test for filestore. Think it's worth it? I wonder if anyone will use that option.

Reply:
testing devices period is an improvement, but it would be smart to also be testing filestore on devices in addition to bluestore.  doesn't have to be done in this PR, but i'd say that we should do it soon.